pandas/util/_decorators.py

```diff
@@ -65,8 +65,9 @@ def deprecate_kwarg(old_arg_name, new_arg_name, mapping=None, stacklevel=2):
     ----------
     old_arg_name : str
         Name of argument in function to deprecate
-    new_arg_name : str
-        Name of preferred argument in function
+    new_arg_name : str | None
+        Name of preferred argument in function. Use none to raise warning that
```

Comment:
none -> None

Reply:
Thanks for catching that.  Please refer to 699805198f7e31e3f6787b612ec507dd71c056a6